cluster/store.go

```diff
@@ -21,3 +32,31 @@ func NewKVStore(ctx context.Context, etcdAddr string) (*KVStore, error) {
 		kv: clientv3.NewKV(c),
 	}, nil
 }
+
+// Get returns the best matched value for the key provided
+func (kvs *KVStore) Get(ctx context.Context, key string) (string, error) {
+	getres, err := kvs.kv.Get(ctx, fmt.Sprintf("%s/%s", storePrefix, key), defaultGetOptions...)
+	if err != nil {
+		return "", fmt.Errorf("failed to get key %s: %w", key, err)
+	}
+
+	if len(getres.Kvs) == 0 {
+		return "", ErrNoKey
+	}
+	return string(getres.Kvs[0].Value), nil
+}
+
+// GetAll returns all values that correspond to the supplied key
+func (kvs *KVStore) GetAll(ctx context.Context, key string) ([]string, error) {
```

Comment:
I think `GetPrefix` would be a better name for this since the difference is that this gets all keys with the same prefix due the the `defaultGetOptions`